src/app/lightning/get-balances.ts

```diff
@@ -23,6 +23,50 @@ export const getTotalBalance = async (): Promise<Satoshis | ApplicationError> =>
   return sumBalances(balances)
 }
 
+export const getInboundBalance = async (): Promise<Satoshis | ApplicationError> => {
+  const offChainService = LndService()
+  if (offChainService instanceof Error) return offChainService
+  const offChainChannelBalances = await offChainService.getInboundOutboundBalance()
+  if (offChainChannelBalances instanceof Error) return offChainChannelBalances
+
+  const inboundOutboundBalance = await Promise.all(
+    offChainService
+      .listActivePubkeys()
+      .map((pubkey) => offChainService.getInboundOutboundBalance(pubkey)),
+  )
+
+  const inbound: Satoshis[] = []
+  inboundOutboundBalance.forEach((balance) => {
+    if ("inbound" in balance) {
+      inbound.push(balance.inbound)
+    }
+  })
```

Comment:
you can replace this by a map 
```suggestion
  const inbound = inboundOutboundBalance.map(b => b.inbound)
```

Reply:
this was not updated, is there a reason to use that notation?

Reply:
the suggestion was causing a type error: https://github.com/GaloyMoney/galoy/pull/2057/commits/b5cb4d345b8549a1242ec53ba9651fb063548b26

Reply:
is because balance could be an error too, so you need to handle it.

```
const inbound = inboundOutboundBalance.map((balance) => {
    if (balance instanceof Error) return toSats(0) 
    return balance.inbound
  })
```